public/docs/dart/latest/_data.json

```diff
@@ -4,66 +4,74 @@
     "title": "Angular Docs",
     "subtitle": "Dart",
     "menuTitle": "Docs Home",
-    "banner": "Current release is <strong>beta.21</strong>. View the <a href='https://github.com/dart-lang/angular2/blob/master/CHANGELOG.md' target='_blank'>change Log</a> to see enhancements, fixes, and breaking changes."
+    "banner": "Angular 2 release is <b>beta.21</b>. View the <a href='https://github.com/dart-lang/angular2/blob/master/CHANGELOG.md' target='_blank'>change log</a> to see enhancements, fixes, and breaking changes."
```

Comment:
I copied that over from the TS side, but I will make those fixes. I created #2407 to help track this change.